docs/services/motion/algorithms.md

```diff
@@ -0,0 +1,39 @@
+---

```

Comment:
Is this page important? @spieswl ? It feels to me like it's geared towards technical users who are curious, but doesn't really matter for someone who just wants to use the Motion Service.

Constraints are linked to from the Move method section where relevant, but yeah I can link to both from the intro if you think these pages are worth calling more attention to?

Reply:
This page is highly technical and all about implementation details, so if we already have a standard practice to make this kind of content prominent, I have no reservations about that. Otherwise, only the most technically savvy will get in here, so I don't believe it is important to highlight.